cmd/kube-apiserver/app/server.go

```diff
@@ -139,6 +140,7 @@ func (s *APIServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.CloudProvider, "cloud_provider", s.CloudProvider, "The provider for cloud services.  Empty string for no provider.")
 	fs.StringVar(&s.CloudConfigFile, "cloud_config", s.CloudConfigFile, "The path to the cloud provider configuration file.  Empty string for no configuration file.")
 	fs.DurationVar(&s.EventTTL, "event_ttl", s.EventTTL, "Amount of time to retain events. Default 1 hour.")
+	fs.StringVar(&s.ClientCAFile, "client_ca_file", s.ClientCAFile, "If set, the file that will be used to secure the secure port of the API server via client cert authentication.")
```

Comment:
Would it be correct to say: "If set, any client presenting a cert signed by the CA in client_ca_file is authenticated with an identity corresponding to the CN of the client cert."

If so, I think that is a clearer flag description.